### PR TITLE
fix(docs/studio): update gitlab self-hosted link

### PR DESCRIPTION
**Pull request by nimdraugsael**
> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

File: content/docs/studio/get-started.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ more projects._
 
    To connect to repositories on your self-hosted GitLab server, you must first
    add a connection to this server and create a team. Refer to the section on
-   [self-hosted GitLab server support](/doc/studio/user-guide/install-github-app)
+   [self-hosted GitLab server support](/doc/studio/user-guide/teams#manage-connections-to-self-hosted-gitlab-servers)
```
Comment (tapadipti) [inline content/docs/studio/get-started.md:63]: Let's point directly to `/doc/studio/user-guide/connect-custom-gitlab-server`; the content in the link you used also points to that one.

**Comment by tapadipti** [comment]
Thanks @nimdraugsael for noticing and fixing this 🙏 
The same issue exists in `/doc/studio/user-guide/projects-and-experiments/create-a-project` also and needs fixing.

**Comment by nimdraugsael** [comment]
@tapadipti thx, fixed in both places

**Review by tapadipti — APPROVED** [review]
Thanks @nimdraugsael

**Pull request merged**